### ISSUE TITLE
Finalize first release

### ISSUE THREAD

**Issue by Jason-Abbott** [jeremychadabbott/jeremy-abbott-music#8]
Possible items you may or may not wish to see for this first release, @jeremychadabbott. I'll either check them off or cross them off.

- [ ] More tracks in album display or more albums
- [ ] Adjust file layout for easier editing
- [ ] Quick link from top down to playlist like [Dean St.](https://www.deanstband.com/)
- [ ] Additional contact information
- [ ] Additional biographical information

The first item, adding more tracks, is the only one I assume you intend. The second item, making it easier to edit, could be done a few ways. I can add more comments into the code or into the README file. The other items didn't strike me as necessary but may be stuff you hoped to see.

**Comment by jeremychadabbott** [comment]
#1 Yes
#2 Comment lines would work fine, I generally understand the code and logic flow and can google what I don't or hit my big bro up.

The rest, yes, I would like to incorporate that but need to set aside some time when I'm bored to put together a bio. for additional contact info, seems repetetive but i have a "jeremy abbott music" FB, twitter and email. What else could anyone want! :)

**Comment by jeremychadabbott** [comment]
To be honest, kinda thought you were mostly done for now and could work on future stuff on an as needed basis. Got me a website that's not embarrassing anymore, actually, quite proud of it! Good foundation / electronic footprint to continue building from. Thank you!

**Comment by Jason-Abbott** [comment]
Yeah, it's pretty well done. Fixing #9 and #10 are the only pressing items. And I'll add more comments to the code so you can find your way to edit on your own.

**Comment by Jason-Abbott** [comment]
BTW, typing number sign and a number is GitHub magic for linking to another issue.
